app/src/main/java/org/simple/clinic/drugs/search/DrugsSearchScreen.kt

```diff
@@ -138,7 +138,7 @@ class DrugsSearchScreen : BaseScreen<
   override fun setDrugSearchResults(searchResults: PagingData<Drug>) {
     val searchQuery = searchQueryEditText.text?.toString().orEmpty()
     drugSearchResultsList.scrollToPosition(0)
-    adapter.submitData(lifecycle, DrugSearchListItem.from(searchResults, searchQuery))
+    adapter.submitData(lifecycle, DrugSearchListItem.from(searchResults, searchQuery, drugFrequencyToFrequencyChoiceItemMap!!))
```

Comment:
So yeah, if we follow the above suggestion where we create this inside the adapter. We can avoid creating a nullable property in the screen.